src/components/ChatBar.js

```diff
@@ -38,12 +38,14 @@ const InputContainer = styled.form`
 `;
 
 function ChatBar({ handleSubmitMessage }) {
+  const [message, setMessage] = React.useState(null);
   return (
     <InputContainer
+      onChange={event => setMessage(event.target.value)}
       onSubmit={event => {
         event.preventDefault();
         if (handleSubmitMessage) {
```

Comment:
Did you try using "try/catch" ? It should work the same and its easier to implement with async functions in the future but its just an idea.

Reply:
I don´t understand what you mean. Why and how would you suggest to implement try/catch?